dissector/udp.md

```diff
@@ -0,0 +1,315 @@
+# BcUDP
+
+This document describes the UDP protocol. UDP, unlike TCP, is lossy, there
```

Comment:
AWESOME docs, nice work

Reply:
Thanks, it was helpful during develop, to have this at hand, as it was too complex for me to remember all the details as I went